file-formats/smtg/type/zob_aff_smtg_v1.intf.abap

```diff
@@ -0,0 +1,156 @@
+interface zob_aff_smtg_v1
+  public.
+
+  " Email Template Metadata --------------------------------------------------------------------------------------------
+  types:
+    "! <p class="shorttext">Administrative Data</p>
+    "! Contains administrative details such as creation and last change information.
+    begin of ty_administrative_data,
```

Comment:
this is already in the header you included in ty_main. This type can be removed.